test/src/config-test.js

```diff
@@ -0,0 +1,27 @@
+describe('td.config', () => {
```

Comment:
I think we should push for "proper" mocha tests, which is to say, not using `=>` for describe/it/before/etc (because it blows away mocha's test context)